packages/manager/modules/iplb/src/iplb-url.constants.js

```diff
@@ -1,7 +1,17 @@
-export const RENEW_URL =
-  'https://www.ovh.com/manager/dedicated/index.html#/billing/autoRenew?selectedType={serviceType}&searchText={serviceName}';
-export const CONTACTS_URL =
-  'https://www.ovh.com/manager/dedicated/index.html#/useraccount/contacts?tab=SERVICES&serviceName={serviceName}';
+export const RENEW_URL = {
+  EU:
+    'https://www.ovh.com/manager/dedicated/#/billing/autoRenew?selectedType={serviceType}&searchText={serviceName}',
+  CA:
+    'https://ca.ovh.com/manager/#/billing/autorenew?selectedType={serviceType}&searchText={serviceName}',
+};
+
+export const CONTACTS_URL = {
+  EU:
+    'https://www.ovh.com/manager/dedicated/#/useraccount/contacts?tab=SERVICES&serviceName={serviceName}',
+  CA:
+    'https://ca.ovh.com/manager/#/useraccount/contacts?tab=SERVICES&serviceName={serviceName}',
```

Comment:
I'm not sure contacts are available in CA

Reply:
It could be enhance into a separate Pull Request.

It will be also reviewed by the new URL bring by the Hub application.

Thanks for mentioning it!